ci/prebuilt/BUILD

```diff
@@ -31,7 +38,8 @@ cc_library(
     name = "event",
     srcs = select({
         ":windows_x86_64": ["thirdparty_build/lib/event.lib"],
-        "//conditions:default": ["thirdparty_build/lib/libevent.a"],
+        ":dynamic_linking": glob(["thirdparty_build/lib*/libevent.so"]),
```

Comment:
It definitely should work the same for those libraries on Linux, *BSD and OS X. Do we support any other platforms?

Reply:
And handling different CPU architectures on RPM-based distros (which use `/usr/lib64` dir instead of `/usr/lib` for 64-bit systems) is the reason why I decided to use `glob` there.

Reply:
Can you comment on how you expect the dynamic libraries to be distributed? As things stand, once you do a dynamic link, the `.so` artifacts that you rely on live in the Bazel cache. How would a user be expected to bundle them up for distribution? Or, is the idea that the user installs these libraries from completely independent sources always (as is generally the case for dynamic deps)? In the latter case, maybe spell this out in the docs.